### ISSUE TITLE
concert page display- concert title duplicated

### ISSUE THREAD

**Issue by brianblain** [jazzto/2015#28]
can we remove/hide the concert title in small text which appears under concert title in big text

**Comment by andrewfn** [comment]
Yes I'll do that. It's actually not the concert title. but a list of the artists. If you look at a concert with multiple artists you'll see, e.g. 
http://torontojazz.com/concert/oscar-petersons-90th-birthday-celebration
But we don't need them since they are repeated below.
